packages/components/package.json

```
@@ -44,7 +42,7 @@
   "types": "lib/index.d.ts",
   "jest": {
     "transform": {
-      "^.+\\.(t|j)sx?$": "babel-jest"
+      "^.+\\.(t|j)sx?$": "ts-jest"
```

`yarn test` 実行するとワーニングが出たのですが、
```
> yarn test
yarn run v1.22.4
$ jest
ts-jest[jest-transformer] (WARN) Got a `.js` file to compile while `allowJs` option is not set to `true` (file: /Users/tkiryu/Documents/repo/frontend/packages/components/.storybook/main.js). To fix this:
  - if you want TypeScript to process JS files, set `allowJs` to `true` in your TypeScript config (usually tsconfig.json)
  - if you do not want TypeScript to process your `.js` files, in your Jest config change the `transform` key which value is `ts-jest` so that it does not match `.js` files anymore
(node:52150) ExperimentalWarning: The fs.promises API is experimental
```
https://github.com/kulshekhar/ts-jest/issues/1102
見ながら以下のように変更したらワーニングは消えました。jsx を処理する必要もないと思うので、これでいいような気がしますがどうでしょう？
```suggestion
      "^.+\\.tsx?$": "ts-jest"
```